graylog2-server/src/main/java/org/graylog2/indexer/GIMMapping.java

```diff
@@ -348,10 +348,14 @@
     private Map<String, Object> dateField() {
         return merge(typeField("date"), Collections.singletonMap(
                 "format",
-                "yyyy-MM-dd HH:mm:ss||yyyy-MM-dd||basic_date_time||basic_date_time_no_millis||epoch_second||date_time_no_millis||date_hour_minute_second_fraction||epoch_millis"
+                dateFormats() + "||basic_date_time||basic_date_time_no_millis||epoch_second||date_time_no_millis||date_hour_minute_second_fraction||epoch_millis"
         ));
     }
 
+    protected String dateFormats() {
+        return "yyyy-MM-dd HH:mm:ss||yyyy-MM-dd";
```

Comment:
Do we need to add the `8` prefix here as well? Or is it not supported in this format?

Reply:
Yes, good catch. Added that and tested it with a GIM index on ES6. :+1: